delta_load_atomicity.md

```diff
@@ -0,0 +1,146 @@
+# Graph delta batch load atomicity
+
+## Rollbacks
+
+Note that some of these options do not specifically enable roll backs, but an update can be rolled
+back (unatomically) by:
+
+* Setting any expirations dates equal to `timestamp - 1` to ∞.
+* Deleting any nodes or edges where the `created` field equals `timestamp`, including merge edges.
+* Setting `last_version` to the prior load version on any nodes or edges where `last_version`
+  equals the load version.
+
+If the `timestamp` is in the future at the time the rollback completes, it's as if the update never
+happened. If not, then any queries between the `timestamp` and the completion of the rollback may
+be spurious.
+
+## Option 1: Reject queries
```

Comment:
We could have a friendly, readable error response message telling them the database is temporarily down, try again soon, etc. to avoid support tickets

Reply:
Done